--- origsrc/libdaemon-0.13/libdaemon/dfork.c	2008-07-28 14:37:56.000000000 -0500
+++ src/libdaemon-0.13/libdaemon/dfork.c	2009-02-16 02:33:01.250000000 -0600
@@ -665,7 +665,7 @@ int daemon_reset_sigs(int except, ...) {
 int daemon_reset_sigsv(const int except[]) {
     int sig;
 
-    for (sig = 1; sig < _NSIG; sig++) {
+    for (sig = 1; sig < NSIG; sig++) {
         int reset = 1;
 
         switch (sig) {
