--- origsrc/libdaemon-0.12/doc/doxygen.conf.in	2005-08-24 11:43:31.000000000 -0500
+++ src/libdaemon-0.12/doc/doxygen.conf.in	2007-07-12 12:42:52.140625000 -0500
@@ -423,7 +423,7 @@
 # directories like "/usr/src/myproject". Separate the files or directories 
 # with spaces.
 
-INPUT                  = ../libdaemon/
+INPUT                  = @top_srcdir@/libdaemon/
 
 # If the value of the INPUT tag contains directories, you can use the 
 # FILE_PATTERNS tag to specify one or more wildcard pattern (like *.cpp 
@@ -461,7 +461,7 @@
 # directories that contain example code fragments that are included (see 
 # the \include command).
 
-EXAMPLE_PATH           = ../examples/
+EXAMPLE_PATH           = @top_srcdir@/examples/
 
 # If the value of the EXAMPLE_PATH tag contains directories, you can use the 
 # EXAMPLE_PATTERNS tag to specify one or more wildcard pattern (like *.cpp 
--- origsrc/libdaemon-0.12/libdaemon/Makefile.am	2007-07-10 12:58:34.000000000 -0500
+++ src/libdaemon-0.12/libdaemon/Makefile.am	2007-07-12 12:42:52.140625000 -0500
@@ -38,4 +38,4 @@
 	daemon.h \
 	$(pkg_include_HEADERS)
 
-libdaemon_la_LDFLAGS = -version-info 3:1:3
+libdaemon_la_LDFLAGS = -version-info 3:1:3 -no-undefined
